src/plasma/plasma_manager.h

```diff
@@ -2,6 +2,7 @@
 #define PLASMA_MANAGER_H
 
 #include <poll.h>
+#include "uthash.h"
```

Comment:
Yeah, the plasma manager is still using uthash and now that plasma.h is not including it any more, we need to include it here.